### DISCUSSION TITLE
Progress - Deadlines - Management

### DISCUSSION THREAD

**Discussion by AntreasAntoniou** [BayesWatch/pytorch-experiments-template#10] [General]
Hello everyone,

I really like this project, but I feel like it's taking too long to progress. I will now make this top of my agenda and get it at a level where we can present our base repo to the group within 10 days from now. I will be giving this 3 hours every day at least. It would be good to know what availability you guys have. I know @learning-luke is quite busy, so I'll let him be more of a commenter/reviewer, so that leaves @jack-willturner. Let me know what availability you have, and we can do some nice pair coding, and get this show on the road ASAP. :) If you are not available, I can go ahead and take care of existing issues, and one of you can review until we reach a state that we all agree with. 

Regards, Antreas

**Comment by learning-luke** [comment]
Hi Antreas, Jack. 

Yeah, had a rough busy week last week and had to prioritise some other work. I am still on board with this and am reviewing all your changes slowly but surely. 

Honestly, I think you guys are both just better at coding up well-structured stuff. However, where can I contribute?

**Reply by AntreasAntoniou** [reply]
I don't mean to push you out or anything! If anything I'd love for you to be part of this as much as possible. I just don't want to overimpose on you, given that you are working hard on your PhD project. :) 

It would be a pleasure to do this with you. I was only looking out for you, I hope my phrasing did not offend! :)

If it did, well, I am sincerely sorry. I should be more careful.

**Reply by learning-luke** [reply]
Not at all, Antreas. You should know by now: if something offends me I'll come right out and say it. 

I literally WANT to contribute because I believe in this. Don't worry!

**Comment by jack-willturner** [comment]
Can't commit to 3 hours per day but happy to spend a chunk of time on this over the next week.

**Reply by AntreasAntoniou** [reply]
Sounds good Jack. ;)
